Dockerfile

```diff
@@ -32,4 +32,4 @@ ADD src/components/k8s/cni/cni-uninstall.sh /cni-uninstall.sh
 ADD src/components/k8s/pcn_k8s/cleanup.sh /cleanup.sh
 ADD src/components/k8s/pcn_k8s/init.sh /init.sh
 
-CMD ["polycubed"]
+CMD ["nsenter","--mount=/host/proc/1/ns/mnt","polycubed"]
```

Comment:
Can you please add a comment saying what is this about?

Reply:
Done.